logic/BaseInstance.h

```diff
@@ -23,7 +23,7 @@
 
 #include "settings/INIFile.h"
 #include "BaseVersionList.h"
-#include "auth/MojangAccount.h"
+#include "minecraft/auth/MojangAccount.h"
```

Comment:
Yay merge artifacts (I really hope this can get into one of the main branches soon :P)